### PR TITLE
Rapid7 release notes

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-05-24-apps_rapid7.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - app catalog
+  - saas cloud
+  - symantec
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of the new Rapid7 app for Sumo Logic. This app leverages Sumo Logic [Cloud-to-Cloud Rapid7 source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source/) to gain deeper insights into asset and vulnerability management activities and prioritize remediation efforts, to minimize attack surfaces, and make well-informed decisions to enhance your overall security posture.
```
Comment (JainM6) [inline blog-service/2023-05-24-apps_rapid7.md:15]:
```suggestion
We're releasing a new app called Rapid7 app that leverages Sumo Logic [Cloud-to-Cloud Rapid7 source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source/). It enables you to obtain a deeper understanding of asset and vulnerability management activities, prioritize remediation efforts, reduce attack surfaces, and make informed decisions to improve your overall security posture.
```

Comment (JainM6) [reply blog-service/2023-05-24-apps_rapid7.md:15]: I've rephrased the para to match the user-friendly tone.

**Review by JainM6 — CHANGES_REQUESTED** [review]
Formatting

File: blog-service/2023-05-24-apps_rapid7.md (hunk 1 above)
Comment (JV0812) [reply blog-service/2023-05-24-apps_rapid7.md]: Thanks @JV0812 . Updated accordingly.

File: blog-service/2023-05-24-apps-rapid7.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Rapid7 (Apps)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - apps
+  - app catalog
+  - saas cloud
+  - symantec
```
Comment (kimsauce) [inline blog-service/2023-05-24-apps-rapid7.md:8]:
```suggestion
  - rapid7
```

**Review by kimsauce — APPROVED** [review]

**Review by kimsauce — APPROVED** [review]

**Pull request merged**